spec/acceptance/openldap__server__dbindex_spec.rb

```diff
@@ -15,6 +15,7 @@ class { 'openldap::server':
         }
         Openldap::Server::Dbindex {
           suffix => 'dc=foo,dc=example,dc=com',
+          require => Openldap::Server::Database['dc=foo,dc=example,dc=com'],
```

Comment:
IIRC this fixed one test case for me. I didn't put it into a seperate commit unfortunately. I'll have to test whether this is always needed.